cidr-filtering-cni

```diff
@@ -0,0 +1,255 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
```

Comment:
btw why not one liner?
set -euo pipefail

Reply:
Yep, Done